Corona-Warn-App/src/test/java/de/rki/coronawarnapp/diagnosiskeys/download/HourPackageSyncToolTest.kt

```diff
@@ -88,8 +90,8 @@ class HourPackageSyncToolTest : CommonSyncToolTest() {
 
         every { downloadConfig.revokedHourPackages } returns listOf(
             RevokedKeyPackage.Hour(
-                day = invalidHour.info.day,
-                hour = invalidHour.info.hour!!,
+                day = invalidHour.info.day.toJavaTime(),
+                hour = LocalTime.of(invalidHour.info.hour!!.hourOfDay, 0),
```

Comment:
Wouldn't it be easier to transform to Java time directly from the data base? We store integers like 23

Reply:
this is only the temporal solution until we use both joda and java time, in the following PR the implementation should look cleaner